lib/jekyll/converters/scss.rb

```diff
@@ -337,6 +339,14 @@ def site_source_relative_from_pwd
         @site_source_relative_from_pwd ||=
           Pathname.new(site_source).relative_path_from(Pathname.new(Dir.pwd)).to_s
       end
+
+      def site_source_url
+        @site_source_url ||= file_path_to_uri("#{site_source}/")
+      end
+
+      def file_path_to_uri(path)
```

Comment:
The subtle difference between `uri` into `url` can create confusion during future refactor.
How about `file_url_from_path` instead?
```suggestion
      def file_url_from_path(path)
```